Projects/Claims/Sources/Models/FlowClaimSingleItemCheckoutStepModel.swift

```diff
@@ -42,7 +42,6 @@ struct ClaimFlowMoneyModel: Codable, Equatable {
 
     func getAmountWithCurrency() -> String {
```

Comment:
This could be a dynamic var as its not taking any arguments :)

Reply:
👍